R/release.r

```diff
@@ -357,3 +362,27 @@ flag_release <- function(pkg = ".") {
   writeLines(msg, file.path(pkg$path, "CRAN-RELEASE"))
   usethis::use_build_ignore("CRAN-RELEASE")
 }
+
+#' Submit a package to a specified location.
+#'
+#' This function is like \code{\link{submit_cran}()} but allows you to specify a
+#' location to submit the package to. It's recommended that you use
+#' \code{\link{release_to}()} rather than this function as it performs more checks
+#' prior to submission.
+#'
+#' @param dest location to submit to as a URL.
+#' @param pkg package description, can be path or package name.  See
+#'   \code{\link{as.package}} for more information
+#'
+#' @export
+submit_to <- function(dest, pkg = ".", args = NULL) {
+  if (yesno("Is your email address ", maintainer(pkg)$email, "?"))
+    return(invisible())
+
+  if (yesno("Ready to submit package to ", dest, "?"))
+    return(invisible())
+
+  pkg <- as.package(pkg)
+  built_path <- build_cran(pkg, args = args)
```

Comment:
Consider renaming `build_cran()` to `build_pkg()`, as the package can be generically built, regardless of destination.